src/dataframes.jl

```diff
@@ -8,7 +8,8 @@ end
 # if it's an array of symbols, set the labels and return a Vector{Any} of columns
 function handle_dfs(df::AbstractDataFrame, d::KW, letter, syms::AbstractArray{Symbol})
     get!(d, :label, reshape(syms, 1, length(syms)))
-    Any[collect(df[s]) for s in syms]
+    ret = Any[collect(df[s]) for s in syms]
+    hcat(ret...)
```

Comment:
It is that the code creates a row Matrix{Vector{Float}} of length `n`, and Plots don't like that - this code makes it a Matrix{Float} with `n` columns, which is what Plots likes.
I'll remind you later if you don't have time to look at it now :-)